pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -278,6 +278,11 @@ type CephVersionSpec struct {
 	// Whether to allow unsupported versions (do not set to true in production)
 	// +optional
 	AllowUnsupported bool `json:"allowUnsupported,omitempty"`
+
+	// ImagePullPolicy describes a policy for if/when to pull a container image
+	// One of Always, Never, IfNotPresent.
+	// +optional
+	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
```

Comment:
```suggestion
	ImagePullPolicy v1.PullPolicy `json:"imagePullPolicy,omitempty"`
```
how about this, it will help when the user has a typo or invalid pullPolicy, during the creation CR it will through an error otherwise in our code we'll try to use the invalid pullPolicy and the deployment will fail.

@travisn @sp98 thoughts?

Reply:
pullPolicy enum would work.